benchmark/src/cli_opt.rs

```diff
@@ -53,9 +45,6 @@ pub struct RubenOpt {
     /// Valid faucet key file path.
     #[structopt(short = "f", long = "faucet_key_file_path", required = true)]
     pub faucet_key_file_path: String,
-    /// Number of accounts to create in Libra.
-    #[structopt(short = "n", long = "num_accounts", default_value = "32")]
-    pub num_accounts: u64,
     /// Number of AC clients.
     /// If not specified or equals 0, it will be set to validator_addresses.len().
```

Comment:
The text here is incorrect based on how the value is set in previous commit

Reply:
Comment updated. Thanks for pointing out.